### ISSUE TITLE
Change Dart SDK version requirement.

### ISSUE THREAD

**Issue by delay** [mezoni/enumerable#1]
I am getting an error when trying to use your package.

The current Dart SDK version is 2.6.0-dev.1.0.flutter-d6c6d12ebf.

Because myApp depends on enumerable any which requires SDK version >=2.6.0-dev.7.0 <3.0.0, version solving failed.

I am on the flutter dev channel currently.  I don't want to switch to the master channel as its not as stable.  Maybe make your package track the stable channel if possible?  Thanks... Looks like a great package but would prefer not to have to fork it to fix the dependency issue...

**Comment by delay** [comment]
Also made a pull request to fix this issue.

**Comment by mezoni** [comment]
I answered you in your pull request.
https://github.com/mezoni/enumerable/pull/2

Short answer: Need to wait until Google Dart developers release 2.6.0 version.

This software was released with the expectation that Google Dart developers will release version 2.6.0 within the specified time.
It seems that they hurried with the announcement (and maybe even will change their plans).

https://github.com/dart-lang/sdk/milestone/75
https://github.com/dart-lang/sdk/commit/c8b903c2f94f57b666d543bba7890c4940f03994#diff-4ac32a78649ca5bdd8e0ba38b7006a1e

It’s not my fault and I can’t change anything. I just relied on publicly available information.
